### DISCUSSION TITLE
insert images in MD files with Markdown Syntax

### DISCUSSION THREAD

**Discussion by dev24hrs** [facebook/docusaurus#9987] [Q&A]
Hi Team,
when i want to insert a full resolution screenshot in md file, only can use `![]()` with md syntax, it's prefect worked, but images still to big.
I refer to the [plugin-ideal-image](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-ideal-image) , but not suitable for me. 
Just want the style like `<img src="" style="zoom:20%;">`. Is there any simpler way to achieve this?
Docusaurus is perfect ,but I'm not familiar with react or js, couldn't find a solution after reading the documentation

**Comment by OzakIOne** [comment]
Here is a small repro of your issue : [stackblitz](https://stackblitz.com/edit/github-sbjuvm?file=docs%2Fintro.md)

You can find more information on Docusaurus documentation [here](https://docusaurus.io/docs/next/markdown-features/assets#static-assets)

I don't think you will be able to achieve what you want without putting your images in `static` folder

**Reply by dev24hrs** [reply]
Yes,i always use imgur for hosting images. When using Typora, I can use `<img src="" style="zoom:50%" />`, but `style="zoom:50%`not worked in Docusaurus, and errors  occurred.
**I cant zoom my images.** 
`style={{ transform: 'scale(0.5)' }}` worked, but is not suitable for me.
Just use `<img src="" width=50%` />
Thanks.
